docs/reference/components/sinks/honeycomb.cue

```diff
@@ -0,0 +1,81 @@
+package metadata
+
+components: sinks: honeycomb: {
+  title: "#{component.title}"
+  short_description: "Batches log events to [Honeycomb][urls.honeycomb] via the [batch events API][urls.honeycomb_batch]."
+  long_description: "[Honeycomb][urls.honeycomb] provides full stack observability—designed for high cardinality data and collaborative problem solving, enabling engineers to deeply understand and debug production software together."
+
+  _features: {
+    batch: {
+      enabled: true
+      common: false,
+      max_bytes: 5242880,
+      max_events: null,
+      timeout_secs: 1
+    }
+    buffer: enabled: true
+    checkpoint: enabled: false
+    compression: enabled: false
+    encoding: enabled: false
+    healthcheck: enabled: true
+    multiline: enabled: false
+    request: {
```

Comment:
Looking at the previous implementation, it does look like we'll have more files to update if we ever change any defaults, as they are no longer inherited from a `request` partial?